src/services/refactors/moveToNewFile.ts

```diff
@@ -533,6 +533,7 @@ namespace ts.refactor {
             case SyntaxKind.ImportEqualsDeclaration:
             case SyntaxKind.ImportSpecifier:
             case SyntaxKind.ImportClause:
+            case SyntaxKind.NamespaceImport:

```

Comment:
I thought about adding a comment given that `NamespaceImport` is in this list but `NamedImports`, but I'm half-guessing that it would be semi-obvious for most people, and I don't know if there's any reference for the list of syntax kinds.

Reply:
I actually don't know why this is - is it because it's somehow handled by `ImportClause`?

Reply:
@DanielRosenwasser, my guess is that one of `NamedImports` or `ImportClause` is for the whole import and the other is for each name in the import, or something similar (since the first is plural).  But I don't want to add a comment with a guess...  I did see that adding `NamedImports` doesn't break any tests, but went with the smaller change.

Reply:
Makes sense, thanks!